spec/requests/families_spec.rb

```diff
@@ -44,6 +89,14 @@
       expect(response).to redirect_to(family_path(family.id))
       expect(request.flash[:notice]).to eql "Family was successfully updated."
     end
+
+    it "renders :edit if the update does not succeed" do
+      put family_path(family), params: { family: { guardian_first_name: nil } }
+
+      expect(response).to have_http_status :ok
+      expect(response).to render_template(:edit)
```

Comment:
Similar to the above comment, I think it might be enough to just test the response code here.